docker-local/dfsp_a/dfsp_a_thirdparty_scheme_adapter.json

```diff
@@ -0,0 +1,71 @@
+{
+  "INBOUND": {
+    "PORT": 4005,
+    "HOST": "0.0.0.0"
+  },
+  "OUTBOUND": {
+    "PORT": 4006,
+    "HOST": "0.0.0.0"
+  },
+  "WSO2_AUTH": {
+    "staticToken": "0706c62f-c022-3c42-8d14-8b5df128876d",
+    "tokenEndpoint": "",
+    "clientKey": "test-client-key",
+    "clientSecret": "test-client-secret",
+    "refreshSeconds": 3600
+  },
+  "REDIS": {
+    "PORT": 6379,
```

Comment:
the port value isn't in sync with docker-compose file - I think it should point out to dfpa-redis port (which by the way isn't specified, so there is a risk that dfspa-thidparty-scheme-adapter will use some other redis instance...)